customIngress default value is wrong type

whenever `.Values.customIngress` is provided, we get the following warning: `coalesce.go:237: warning: skipped value for artifactory.customIngress: Not a table.`

For example:
```
$ helm template --set customIngress.foo=bar --set ingress.enabled=true --show-only=templates/ingress.yaml --version 107.111.12 jfrog/artifactory
coalesce.go:237: warning: skipped value for artifactory.customIngress: Not a table.
coalesce.go:237: warning: skipped value for artifactory.customIngress: Not a table.
---
# Source: artifactory/templates/ingress.yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: release-name-artifactory
  labels:
    app: artifactory
    chart: artifactory-107.111.12
    release: release-name
    heritage: Helm
spec:
  defaultBackend:
    service:
      name: release-name-artifactory
      port:
        number: 8082
  rules:
---
# Source: artifactory/templates/ingress.yaml
foo: bar
```

this is because it's taking the default value, `customIngress: ""` and trying to merge that in with the map value provided at the command line (or in a values file, etc).

## Versions
* jfrog/artifactory chart 107.111.12
* helm 3.17.4

--- a/stable/artifactory/CHANGELOG.md
+++ b/stable/artifactory/CHANGELOG.md
@@ -1,6 +1,8 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
+* Fix default value for `customIngress` to address helm warnings. [GH-2020](https://github.com/jfrog/charts/issues/2020)
+
 ## [107.117.14] - August 05, 2025
 * Added support for AWS S3 V3 credentials (identity and credential) from a Kubernetes Secret
 * Added support for Azure credentials (accountName and accountKey) from a Kubernetes Secret

--- a/stable/artifactory/values.yaml
+++ b/stable/artifactory/values.yaml
@@ -139,7 +139,7 @@ ingress:
   ## This is an experimental feature, enabling this feature will route all traffic through the Router.
   disableRouterBypass: false
 ## Allows to add custom ingress
-customIngress: ""
+customIngress: {}
 ## There is a need to separate HTTP1.1 traffic from gRPC (HTTP2) to benefit
 ## 1. Use the Nginx keepalive for HTTP1.1
 ## 2. Keep gRPC over the HTTP2 connections only

